modules/react/loading-dots/stories/examples/DarkBackgrounds.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {LoadingDots, loadingDotsStencil} from '@workday/canvas-kit-react/loading-dots';
+import {system} from '@workday/canvas-tokens-web';
+import {createStyles, createStencil} from '@workday/canvas-kit-styling';
+
+const styleOverrides = {
+  parentContainer: createStyles({
+    display: 'flex',
+    gap: system.space.x4,
+  }),
+};
+
+const loadingStencil = createStencil({
+  base: {
+    borderRadius: system.shape.round,
+    backgroundColor: system.color.bg.overlay,
+    height: 80,
+    width: 80,
+    alignItems: 'center',
+    justifyContent: 'center',
+    display: 'flex',
+    [loadingDotsStencil.vars.loadingDotColor]: system.color.bg.alt.strong,
```

Comment:
I think by default this is the color of the loading dots, so no need to showcase that here.

Reply:
my bad, this is supposed to be `system.color.icon.inverse` for the circle variant design!